components/brave_vpn/common/wireguard/wireguard_utils.cc

```diff
@@ -50,6 +53,14 @@ absl::optional<std::string> CreateWireguardConfig(
   return config;
 }
 
+wireguard::WireguardKeyPair GenerateNewX25519Keypair() {
+  crypto::EnsureOpenSSLInit();
+  uint8_t pubkey[32] = {}, privkey[32] = {};
+  X25519_keypair(pubkey, privkey);
```

Comment:
👍🏻

Reply:
lol @spylogsster, this was a very condense way of saying no concerns with using this crypto implementation (which we use elsewhere anyways)